### DISCUSSION TITLE
Android BroadcastReceiver will trigger the button RelayCommand

### DISCUSSION THREAD

**Discussion by YSW33** [AvaloniaUI/Avalonia#15462] [Q&A]
Every time it triggers my button

**Comment by timunie** [comment]
Can you be more precise pls?

**Reply by YSW33** [reply]
I have an interface,ListBox.ItemTemplate is UniformGrid,UniformGrid.Childrens is IconButton。
I ordered an Android message, which came from infrared scanning。
When I clicked the button and a message popped up, using Android infrared scanning triggered the button's Command。
I am using CommunityToolkit.Mvvm。

**Comment by YSW33** [comment]
test code 
[BAP.Universal.Desktop.zip](https://github.com/AvaloniaUI/Avalonia/files/15071046/BAP.Universal.Desktop.zip)
1
![image](https://github.com/AvaloniaUI/Avalonia/assets/141573398/59ef7008-10e1-4a0e-ab8b-04b3f1f7e911)
2
![image](https://github.com/AvaloniaUI/Avalonia/assets/141573398/047f41bf-a785-4bbf-a138-7630124b8bdc)
3 
![image](https://github.com/AvaloniaUI/Avalonia/assets/141573398/45bea812-e5b0-429f-800a-db881ad91366)
Using PDA scanning，Will continuously trigger  ModuleViewModel.OnItemActivate

**Reply by timunie** [reply]
We cannot debug an entire App for you. Either a minimal sample is required or you can request paid support here. Thank you for your kind understanding.
